src/Symfony/Bundle/WebProfilerBundle/Resources/config/schema/webprofiler-1.0.xsd

```diff
@@ -8,7 +8,14 @@
     <xsd:element name="config" type="config" />
 
     <xsd:complexType name="config">
-        <xsd:attribute name="toolbar" type="xsd:boolean" />
+        <xsd:sequence>
+            <xsd:element name="toolbar" type="toolbar" />
+        </xsd:sequence>
         <xsd:attribute name="intercept-redirects" type="xsd:boolean" />
     </xsd:complexType>
+
+    <xsd:complexType name="toolbar">
+        <xsd:attribute name="enabled" type="xsd:boolean" />
+        <xsd:attribute name="ajax_replace" type="xsd:boolean" />
```

Comment:
XML attributes use dashes, not underscores (the Config component normalizes that when processing the config)